beeswarm/server/db/database_actor.py

```diff
@@ -391,13 +393,14 @@ def _classify_malicious_sessions(self):
             db_session.commit()
 
         # find and process honeypot sessions that did not get classified during persistence.
-        sessions = db_session.query(Session).filter(Session.discriminator == None) \
+        sessions = db_session.query(Session, Drone.name).filter(Session.discriminator == None) \
```

Comment:
Would be nice to rename the `sessions` variable, since it contains the drone name too now.